requirements.txt

```diff
@@ -25,5 +25,5 @@ hjson  # for reggen
 
 # For peakrdl
 peakrdl
-peakrdl-rawheader @ git+https://github.com/micprog/peakrdl-rawheader.git
+peakrdl-rawheader @ git+https://github.com/colluca/PeakRDL-rawheader.git@7b8dbc9ad5854dc1cdaf36d4ea024c29ffb00a4c
```

Comment:
It might make sense to move this plugin to a PULP repository if the features cannot be merged in the upstream `peakrdl` repository

Reply:
Yeah the features will not be merged upstream, the author suggested that this be kept as a plugin. If we want to move it to a PULP repo we should talk to @micprog about it, but I wouldn't wait on this to merge the PR.